benchmarks/README.md

```diff
@@ -143,13 +145,13 @@ h2o groupby query 1 took 1669 ms
 [1]: http://www.tpc.org/tpch/
 [2]: https://www1.nyc.gov/site/tlc/about/tlc-trip-record-data.page
 
-## Parquet filter pushdown benchmarks
+## Parquet benchmarks
 
-This is a set of benchmarks for testing and verifying performance of parquet filter pushdown. The queries are executed on
-a synthetic dataset generated during the benchmark execution and designed to simulate web server access logs.
+This is a set of benchmarks for testing and verifying performance of parquet filtering and sorting.
+The queries are executed on a synthetic dataset generated during the benchmark execution and designed to simulate web server access logs.
 
 ```base
-cargo run --release --bin parquet_filter_pushdown --  --path ./data --scale-factor 1.0
+cargo run --release --bin parquet --  --path ./data --scale-factor 1.0
```

Comment:
🤔  I tried this command and got:

```shell
alamb@MacBook-Pro-8:~/Software/arrow-datafusion$ CARGO_TARGET_DIR=/Users/alamb/Software/target-df cargo run --release --bin parquet -- --path ./data --scale-factor 1.0
error: no bin target named `parquet`.
Available bin targets:
    print_config_docs
    tpch
    h2o
    nyctaxi
```

I think it is because the file was moved out of `benchmarks/src/bin` and into `benchmarks/parquet.rs`

Reply:
yes that's my bad 😅